nix/modules/lanzaboote.nix

```diff
@@ -161,6 +168,52 @@ in
       '';
       default = [ ];
     };
+
+    allowUnsigned = lib.mkEnableOption "" // {
```

Comment:
This should be the default for `autoGenerateKeys.enable = true;`

Otherwise you get:

```
Failed to install generation 22: Get stub name: Failed to read public key from /etc/secureboot/keys/db/db.pem: No such file or directory (os error 2)
Failed to install bootloader
Command 'systemd-run -E LOCALE_ARCHIVE -E NIXOS_INSTALL_BOOTLOADER --collect --no-ask-password --pipe --quiet --service-type=exec --unit=nixos-rebuild-switch-to-configuration /nix/store/zpzjaawd1hqp70h56r3vbmkal6wn8akl-nixos-system-ms-a2-26.05.20251130.2d293cb/bin/switch-to-configuration boot' returned non-zero exit status 1.
```